lfs/localstorage.go

```diff
@@ -0,0 +1,46 @@
+package lfs
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/github/git-lfs/vendor/_nuts/github.com/rubyist/tracerx"
+)
+
+func ClearTempObjects() {
+	filepath.Walk(LocalObjectTempDir, func(path string, info os.FileInfo, err error) error {
+		if shouldDeleteTempObject(path, info) {
+			return os.RemoveAll(path)
```

Comment:
I think you should always return no error, otherwise a failure to delete one file will stop deletion of the rest. Stray locks on files can cause issues like this (especially on Windows). Maybe just trace any errors.